### ISSUE TITLE
train question

### ISSUE THREAD

**Issue by liu15509348793** [ultralytics/ultralytics#17062]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello, thank you for your work, I am currently experiencing a problem, my GPU can only run continuously for 72 hours at most, but I have not finished training, I want to use a power failure to continue training, what should I do, how to set the parameters, can you help me, thank you very much

**Comment by liu15509348793** [comment]
> You can resume the training by loading the `last.pt` and then just running `model.train(resume=True)`.

If I do this, can the learning rate continue with the original learning rate? Because I found that the loss dropped extremely after doing this, I suspect that the learning rate is not used in the original, I have a total of 500 epochs, and the last few epochs are then trained, the picture is as follows
![results](https://github.com/user-attachments/assets/0ffa80ae-0c57-4ef3-aaa3-666bec1f2648)

**Comment by Y-T-G** [comment]
The learning rate is resumed from the same point. You can verify it in the `results.csv` file. But there can be differences in augmentations because the randomness state is not restored.

**Comment by liu15509348793** [comment]
> The learning rate is resumed from the same point. You can verify it in the `results.csv` file. But there can be differences in augmentations because the randomness state is not restored.

ok,I see,Thank you very much

**Comment by liu15509348793** [comment]
> The learning rate is resumed from the same point. You can verify it in the `results.csv` file. But there can be differences in augmentations because the randomness state is not restored.

Also, I see val loss, does the val process still calculate loss? What does this do?

**Comment by Y-T-G** [comment]
Which one?

**Comment by liu15509348793** [comment]
> Which one?

Are these three of the following the losses calculated at val? What does it do? Backpropagation should not be based on the loss of the training set
![Uploading QQ截图20241025093740.png…]()

**Comment by liu15509348793** [comment]
> Which one?

![Uploading 111.png…]()

**Comment by liu15509348793** [comment]
> Which one?

That is, val/box_loss, val/cls_loss, val/dfl_loss, these three items are the losses calculated at val? What does it do? Backpropagation should not be based on the loss of the training set

**Comment by Y-T-G** [comment]
They are used for plotting and to check overfitting.
